projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -169,28 +402,46 @@ export function navigateToQuoteListFromQuoteDetails() {
 
 /**
  * Checks the displayed quote, assuming that it is in draft state.
- * @param meetsThreshold does the quote meet the threshold
- * @param productId product id of a product which is part of the quote
+ * @param meetsThreshold Does the quote meet the threshold
+ * @param productId Product id of a product which is part of the quote
  */
 export function checkQuoteInDraftState(
   meetsThreshold: boolean,
   productId: string
 ) {
-  checkQuoteState('Draft');
-  this.checkGlobalMessageDisplayed(!meetsThreshold);
-  this.checkSubmitButton(meetsThreshold);
-  cy.get('.cx-code').should('contain.text', productId);
+  log(checkQuoteInDraftState.name);
```

Comment:
I would suggest not to log the name of the method but to log the method description, for instance:
> Verifies the displayed quote

Reply:
done